README.md

```diff
@@ -42,7 +42,19 @@ Externals are not bundled when packaging and publishing the component, for bette
  
 ### features
 - `Server Side Rendering` = server side rendering should work as for any other OpenComponent. Differently from the 'oc-template-react' template this template rely on [`renderToStaticMarkup()`](https://reactjs.org/docs/react-dom-server.html#rendertostaticmarkup) for SSR as no hidration is needed.
-- `css` is supported by converting any css imported into [Style Objects](https://www.w3schools.com/jsref/dom_obj_style.asp). This object is compatible with React Inline Styles. 
+- `css` is supported by converting any css imported into [Style Objects](https://www.w3schools.com/jsref/dom_obj_style.asp). This object is compatible with React Inline Styles.
```

Comment:
👋 is this because emails require everything to be in inline styles?

Reply:
Yes, this repo is still a WIP but it's a fully opinionated OC template to work with e-mails, therefore SSR with staticMarkup rendering, inlined styles (with possibility of style tag for media queries), etc. 👍 

The goal is to have all the transactional emails in react very soon and capable of fully integrating to https://github.com/opentable/design-tokens
Long term there may be opportunities to share React components, but this is not a priority on the short term.
What do you think?